finagle-redis/src/test/scala/com/twitter/finagle/redis/commands/sorted-set/SortedSetClientIntegrationSuite.scala

```diff
@@ -12,6 +15,8 @@ import org.scalatest.junit.JUnitRunner
 @RunWith(classOf[JUnitRunner])
 final class SortedSetClientIntegrationSuite extends RedisClientTest {
 
+  val TIMEOUT = 5.seconds
```

Comment:
I really like that you've added a timeout: it seems it has been omitted from the other test cases in the suite.